health/health.d/memory.conf

```diff
@@ -28,7 +31,11 @@ component: Memory
     every: 1m
      crit: $this > 0
     delay: down 1h multiplier 1.5 max 1h
-     info: number of ECC uncorrectable errors in the last 10 minutes
+     info: Number of ECC uncorrectable errors in the last 10 minutes. \
+           There are ECC uncorrectable errors. This is a fatal issue. \
+           While the error may be due to a failing DRAM chip, \
+           it can also be caused by incorrect seating or improper contact between the socket and DIMM. \
```

Comment:
Same as above, this may be a transient error due to comsic rays.

Reply:
We took this info from [Intel's System EventLog Troubleshooting Guide](https://www.intel.com/content/dam/support/us/en/documents/server-products/SystemEventLog_TroubleshootingGuide.pdf).

<img width="756" alt="Screenshot 2021-09-15 at 12 53 07" src="https://user-images.githubusercontent.com/22274335/133412273-1494eef0-103b-45f0-ac1c-584de11f19e8.png">

---

What do you suggest?

Reply:
> It may also just be a transient error due to cosmic rays.

Nice to know, but I think it is ok to not mention it (unless NASA uses Netdata).